src/bidiMapper/domains/script/ScriptProcessor.ts

```diff
@@ -51,6 +52,7 @@ export class ScriptProcessor {
   async addPreloadScript(
     params: Script.AddPreloadScriptParameters
   ): Promise<Script.AddPreloadScriptResult> {
+    const contexts = new Set<BrowsingContextImpl>();
```

Comment:
Be careful here: The intention of the `Set` is likely to deduplicate contexts, but "BrowsingContextImpl" is an opaque object. The deduplication would only happen if it was e.g. via context id (a simple string).

Side effect: Let's add one test that intentionally passes the same context twice, and verify that the script is only performed once. Something like this:

```
.addPreloadScript(script="window.count = (window.count ?? 0) + 1", contexts=[context_id, context_id]);

Expected: I _think_ it should be equal to 1, as it should be deduplicated. But the way the current implementation is done I believe it would equal 2.
```